### DISCUSSION TITLE
https://zerofiltre.tech/cours/1-mettez-enfin-en-place-le-domain-driven-design/1-qu-est-ce-que-le-ddd-en-r%C3%A9alit%C3%A9

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#256] [Announcements]
# https://zerofiltre.tech/cours/1-mettez-enfin-en-place-le-domain-driven-design/1-qu-est-ce-que-le-ddd-en-r%C3%A9alit%C3%A9

des petites bonnes actions.

https://zerofiltre.tech/cours/1-mettez-enfin-en-place-le-domain-driven-design/1-qu-est-ce-que-le-ddd-en-r%C3%A9alit%C3%A9

**Comment by imphilippesimo** [comment]
Clair et limpide !
